scripts/helpers/createExtensionManifest.js

```diff
@@ -771,6 +771,10 @@ const createExtensionManifest = ({ version }) => {
             },
             transforms: {
               type: "object"
+            },
+            customCode: {
+              type: "string",
```

Comment:
this will throw an error for existing `update variable` actions without `customCode`